code/modules/mob/living/silicon/robot/robot_model.dm

```diff
@@ -578,6 +577,11 @@
 	if(lube)
 		lube.reagents.add_reagent(/datum/reagent/lube, 2 * coeff)
 
+	var/obj/item/soap/cyborg/soap = locate(/obj/item/soap/cyborg) in basic_modules
+	if(soap)
+		if(soap.uses < 300)
```

Comment:
Rather then checking to see if something is true and indenting, consider checking to see if the inverse is true, and then returning.

S called early returns, keeps indentation levels to a readable minimum, and helps make logic more clear
https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#use-early-returns